presto-spi/src/main/java/com/facebook/presto/spi/PageBuilder.java

```diff
@@ -28,7 +28,7 @@
 
 public class PageBuilder
 {
-    private static final int DEFAULT_INITIAL_EXPECTED_ENTRIES = 1024;
+    private static final int DEFAULT_INITIAL_EXPECTED_ENTRIES = 64;
```

Comment:
It probably makes sense to reduce:
`com.facebook.presto.sql.analyzer.FeaturesConfig#filterAndProjectMinOutputPageRowCount`
too

Reply:
We expect this to reduce the memory size, but it would still be nice to see the impact of this change on performance (maybe using the benchmarks in presto-benchmark) and memory usage/gc behavior etc. (may need to write a new benchmark).